modules/user/user.resolver.js

```diff
@@ -1,26 +1,39 @@
 const userService = require('./user.service');
+const { uploadFiles } = require('../upload/upload.service');
 
 const userQuery = {
   getAllUsers: () => userService.getAllUsers(),
-  getUserByToken: (parent, args, context) => {
-    try {
-      return context.user;
-    } catch ({ message }) {
-      return {
-        statusCode: 404,
-        message,
-      };
-    }
-  },
+  getUserByToken: (parent, args, context) => context.user,
   getUserById: (parent, args) => userService.getUser(args.id),
 };
 const userMutation = {
   registerUser: (parent, args) => userService.registerUser(args.user, args.language),
   loginUser: (parent, args) => userService.loginUser(args.loginInput),
   loginAdmin: (parent, args) => userService.loginAdmin(args.loginInput),
   deleteUser: (parent, args) => userService.deleteUser(args.id),
-  updateUserById: (parent, args) => userService.updateUserById(args.user, args.id),
-  confirmUser: (parent, args) => userService.confirmUser(args.token),
+  updateUserById: async (parent, args) => {
+    try {
+      if (!args.upload) {
```

Comment:
All of this logic should be inside the service, not in the controller

Reply:
removed